tests/microbenchmarks.py

```diff
@@ -203,11 +232,14 @@ def test_dict_set(test_case):
             dict["meow"] = 4
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """),
+        timed=True)
+
 
 def test_class_init(test_case):
-    print("Running" , "test_class_init")
-    test_case.runAsJava(adjust("""
+    print("Running", "test_class_init")
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils